causalpy/plot_utils.py

```diff
@@ -13,28 +14,46 @@ def plot_xY(
     ax: plt.Axes,
     plot_hdi_kwargs: Optional[Dict[str, Any]] = None,
     hdi_prob: float = 0.94,
+    label: Optional[str] = "",
```

Comment:
Remember `Optional[str]` is equivalent to `Union[str, None`. I think we probably do not want this so we should remove `Optional`. See https://github.com/pymc-labs/CausalPy/pull/117

Reply:
Changed to `Union[str, None]`. This also allowed me to remove a kwarg